### PR TITLE
fixes for manual correction

### LINKED ISSUE #32: Manual labelling correction
When vertebral labelling works, each vertebra is labelled. However, manual correction of labelling currently uses spine-generic that requests user to manually input only 2 labels. How should i process labelling using the manual correction that contains only 2 labels?
Initial comment:
https://github.com/sct-pipeline/csa-atrophy/pull/31#issuecomment-670941179

### PR DESCRIPTION
**Pull request by PaulBautin**
Up to now shell script for manual correction was basic and did not offer very many options. With the use of a modified version of `manual_correction.py` from spine_generic https://github.com/spine-generic/spine-generic/blob/master/spinegeneric/cli/manual_correction.py csa-atrophy is now capable of correcting all files (labeling, and spinal cord segmentation). Moreover the use of spine generic allows a better organization of derivative files using BIDS convention. 

For the use of spine_generic `manual_correction.py` script, modifications were made:
- `csa_atrophy_results/results` folder is organized with one anat directory for each rescaling. `manual_correction.py` function `check_file` was modified to be able to find files.
- User is now asked with sct_label_utils to enter disks from 2 to 6 in the spinal cord. disk 2 ==> C2-C3

DONE
- import and modify `manual_correction.py` from spine_generic
- update process_data.sh
- update setup.py

FIX #32 #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ python csa_rescale_stat.py -i csa_atrophy_results/results/csa_data -o csa_atroph
...
-./manual_labeling_correction.sh
+manual_correction -config config_correction.yml -path-in csa_atrophy_results/results -path-out data
 ~~~
-The bash script outputs all effectuated manual labelings to 'results_corrected/seg_manual'.
-It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'seg_manual', otherwise labeling will be done automatically.
+The bash script outputs all effectuated manual corrections to 'data/derivatives/labels'.
+It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'data/derivatives/labels', otherwise labeling will be done automatically.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
+sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual data/derivatives/labels -script process_data.sh
```
Comment (jcohenadad) [inline README.md:60]: data/derivatives/labels should probably be removed

**Comment by jcohenadad** [comment]
pls check your indentation, stick with one convention

File: README.md (hunk 1 above)
Comment (PaulBautin) [reply README.md:60]: done, already present in process_data,

Comment (PaulBautin) [reply README.md:60]: actually it was a mistake, i've renamed it "output directory". It is actually not a path but a directory name.

**Comment by PaulBautin** [comment]
I will close without merging this branch because it is outdated.

**Pull request closed without merging**